### DISCUSSION TITLE
Retain pane zoom state on pane switch

### DISCUSSION THREAD

**Discussion by alanfzf** [wezterm/wezterm#4062] [Q&A]
Sorry couldn't find a way to toggle this option in the documentation, but how i can make wez term retain pane zoom state when switching to other panes, i want it to behave more like windows terminal, here's an example of what i mean:

https://github.com/wez/wezterm/assets/29936954/3c2e2528-295a-41a8-9782-a3921e43279a

edit: I'm aware of [unzoom_on_switch_pane](https://wezfurlong.org/wezterm/config/lua/config/unzoom_on_switch_pane.html) but this afaik only disables the ability to switch panes when zoomed in.

**Comment by wez** [comment]
wezterm doesn't have that feature/function

**Reply by wez** [reply]
but: you could use [Multiple](https://wezfurlong.org/wezterm/config/lua/keyassignment/Multiple.html) to combine unzoom, pane switching and zooming in your own key assignment.
